includes/classes/InternalConnections/NetworkSiteConnection.php

```diff
@@ -144,7 +144,7 @@ public function push( $post_id, $args = array() ) {
 			 * @return {bool} If Distributor should push the post media.
 			 */
 			if ( apply_filters( 'dt_push_post_media', true, $new_post_id, $media, $post_id, $args, $this ) ) {
-				\Distributor\Utils\set_media( $new_post_id, $media );
+				\Distributor\Utils\set_media( $new_post_id, $media, [ 'use_filesystem' => true ] );
```

Comment:
The PR works in my test. But I think we only use filesystem for network connection, so we can use `DT_IS_NETWORK` to detect multisite installations without introducing a new argument for `set_media` function.

Reply:
@petenelson What do you think about my comment above? (Forgot to tag you 😅)

Reply:
@dinhtungdu So I think the only issue using that constant here is that is only set to true if we are on multisite and if Distributor is network activated. So there's a chance someone could be using multisite but not have the plugin network activated, just activated on the individual sites they want to use. In that case, using that constant wouldn't work